CLI/other/man/get.txt

```diff
@@ -0,0 +1,42 @@
+NAME
+    get - get object information
+
+SYNOPSIS
+    get [OPTION]... [PATH] [FILTERS]
+
+DESCRIPTION
+    Retrieves object information from API and displays it's information in JSON format.
+
+    If no PATH is given, then the current path will be used (equivalent to "get .").
+    PATH can use the character * to use wildcards. A PATH equal to * means all objects in the current path while A* means all object which name starts with A.
+
+    [FILTERS] is an optional list of filters that can be used to only get the objects that meet certain conditions.
+    In order to use FILTERS, PATH must be present.
+    The format to use is attribute1=expected_value,attribute2=expected_value,....
+
+OPTIONS
+    -r
+        Specifies that you want to do a recursive get, i.e., not only get objects in the current path but the entire descendant hierarchy.
+        Using -r with an exact PATH has no effect.
```

Comment:
I would expect to get the whole descendant hierarchy

Reply:
What did you find strange? Doing local tests both cases seem to work fine.

Anyway, if this part of the documentation was wrong, fixed it in 65e05309.